### ISSUE TITLE
handle null in getTemplateFromKnexPOJO

### ISSUE THREAD

**Issue by kevinhuang12345678** [haven-life/persistor#56]
in PersistObjectTemplate.getTemplateFromKnexPOJO, if value=null for isRemoteDoc, it logs:
ERROR: query[getTemplateFromKnexPOJO]: (__amorphicContext={} component="persistor" module="query" activity="getTemplateFromKnexPOJO" data="there was a problem. remote object key must be a string")

it shouldn't log out the Error.

**Comment by srksag-zz** [comment]
@nsaenz7 - I think this is S3 implementation. Do you want to look into this.

**Comment by kevinhuang12345678** [comment]
yes. it is for S3 implementation. i can look into it. should i create pull request from here? the latest version is 5.5.0?
